docs/user-guide/rules.md

```diff
@@ -310,17 +308,141 @@ For example, specifying if a single space or no space must come after a comma in
 - `function-comma-space-after`: `string` - `"always"|"never"`
 
 ```css
-    a { transform: translate(1, 1) }
-/**                           ↑
- *  The space after this commas */
+a { transform: translate(1, 1) }
+/**                       ↑
+ * The space after this commas */
 ```
 
 The plural of the punctuation is used for `inside` rules. For example, specifying if a single space or no space must be inside the parentheses of a function:
 
 - `function-parentheses-space-inside`: `string` - `"always"|"never"`
 
 ```css
-    a { transform: translate( 1, 1 ); }
-/**                         ↑      ↑
+a { transform: translate( 1, 1 ); }
+/**                     ↑      ↑
  * The space inside these two parentheses */
 ```
+
+## Rules work together
```

Comment:
I wonder if we should split all this stuff at the bottom of the rule list into a separate doc, "About rules". I bet it's lost down there at the bottom of the giant list.